universal-application-tool-0.0.1/app/repository/ProgramRepository.java

```diff
@@ -23,8 +23,8 @@ public ProgramRepository(EbeanConfig ebeanConfig, DatabaseExecutionContext execu
   }
 
   /** Return all programs in a set. */
-  public CompletionStage<Set<Program>> listPrograms() {
-    return supplyAsync(() -> ebeanServer.find(Program.class).findSet(), executionContext);
+  public CompletionStage<List<Program>> listPrograms() {
```

Comment:
question: Why the change to list instead of set? I tend to prefer sets if there is no strong reason for maintaining order

nit: If we decide to go with a list, please update comment

Reply:
I changed it because the method name has "list".